src/firebase_rest_to_rust.rs

```diff
@@ -37,8 +37,15 @@ pub(crate) fn firebase_value_to_serde_value(v: &dto::Value) -> serde_json::Value
         }
     } else if let Some(map_value) = v.map_value.as_ref() {
         let mut map: Map<String, serde_json::value::Value> = Map::new();
-        for (map_key, map_v) in &map_value.fields {
-            map.insert(map_key.clone(), firebase_value_to_serde_value(&map_v));
+        match &map_value.fields {
```

Comment:
Maybe use `if let Some(...)` here?

Reply:
Sure thing - I'll make this change in another commit